cypress/integration/CreatePfLpa.feature

```diff
@@ -1,12 +1,32 @@
-Feature: Create an LPA
+Feature: Create a Property and Finance LPA
+
+    I want to create a Property and Finance LPA
+
+    Background:
+        Given I ignore application exceptions
  
-  I want to create an LPA
+    #    @focus
+    Scenario: Dashboard has Link to Type page
+        # we use seeded user here because a newly signed-up user would not yet have a dashboard page
+        Given I log in as seeded user
+        When I click "createnewlpa"
+        Then I am taken to the lpa type page
   
     @focus
-    Scenario: Create LPA
-        # todo: during development, using seeded user saves time, but
-        # to avoid tests having external dependency on seeded user,
-        # the following should ultimately change to logging in with a
-        # "seeded-by-code" newly signed up standard user
-    Given I log in as seeded user
-    Then I am taken to the type or dashboard page
+    Scenario: Create LPA with error first
+        Given I log in as appropriate test user
+        Then I visit the type page
+        When I click "save"
+        Then I see in the page text
+            | There was a problem submitting the form |
+            | You need to do the following: |
+            | Choose a type of LPA |
+        Then I choose Property and Finance
+        When I click "save"
```

Comment:
Similarly, I would probably just add a step to check the page loads after this and doesn't just hang

Reply:
Have added the next step which is that we're on the donor page.